CVE-2022-4421.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Expected location (2022/4xxx/CVE-2022-4421.json) of file does not match actual location (CVE-2022-4421.json)